sqlvm-alwayson-cluster/README.md

```diff
@@ -46,3 +42,29 @@ This template creates the following resources:
 	+ 	Witness - Latest Windows Server 2012 R2 Image
 
 + 	Once deployed access can be gained by the public IP address of Primary Domain Controller
+
+## Deployment steps
+
+You can click the "deploy to Azure" button at the beginning of this document.
+
+|Name|Description|Example|
```

Comment:
@ninarn The table is going to cause a problem when Azure.com starts rendering the complete readme file on Azure.com. There will be two tables for your templates.